cognite/client/_api/datapoints.py

```diff
@@ -22,6 +24,9 @@ def __init__(self, *args, **kwargs):
         self._DPS_LIMIT = 100000
         self._POST_DPS_OBJECTS_LIMIT = 10000
         self._RETRIEVE_LATEST_LIMIT = 100
+        self.synthetic = SyntheticDatapointsAPI(
```

Comment:
Doesn't this mean it will be exposed on the CogniteClient from the non-experimental namespace?